salt/iiif/config/etc-nginx-sites-enabled-loris.conf

```diff
@@ -22,8 +17,13 @@ server {
     {% endif %}
 
     {% if salt['elife.only_on_aws']() %}
-    server_name {{ salt['elife.cfg']('project.project_hostname') }}
-                {{ salt['elife.cfg']('project.full_hostname') }};
+
+    # order is significant as the first is used when redirecting requests
```

Comment:
these were introduced when proxying http -> http with the uWSGI `http-socket` and may no longer be necessary.